docs/migrate-to-ory/migrate/migrate-strategies.mdx

```diff
@@ -0,0 +1,116 @@
+- You need to retire the current solution soon
+- Your legacy IAM vendor restricts your ability to use a graceful identity migration strategy (for example, Auth0).
+
+##### Advantages of bulk identity migration
+
+- Simplicity: Since there is only one "go-live", the identity migration process is easier to manage and plan.
+- Time efficiency: The identity migration process happens in one go, reducing the transition time.
+- Less complexity: There is no need to run two systems in parallel and you can retire the previous solution immediately after
+testing and validating the migration.
+
+##### Drawbacks of bulk identitiy migration
+
+- High risk: If any issues occur during the cutover, the impact can be significant, affecting all users. If might be difficult
+to perform a rollback.
+- Downtime: This approach may require planned system downtime to ensure data consistency, which can disrupt users.
+- Increased preparation: Requires extensive planning and testing to mitigate risks, making it more resource-intensive during that
+  phase.
+
+### Graceful identity migration
+
+Graceful identity migration - also called "automatic", "trickle", "just-in-time", or "online" migration - involves running both the old and
+new systems in parallel, gradually migrating user identities as the user authenticates. This approach features two "go-lives": the initial
+application "go-live" and subsequent user-specific cutovers during login.
+
+#### When to use graceful identity migration
+A graceful identity migration is a good choice when:
+
+- You don't have access to hashed credentials or they are hashed with a proprietary algorithm
+- Authentication system downtime is not acceptable
+- Running the current solution until the end of migration isn't a problem
+- You need a smooth user experience, so your users will not notice the migration
+
+##### Advantages of graceful identity migration
+
+- Low risk: The gradual identity migration reduces the risk of widespread issues, as only a few users are affected at any given time.
+- No hashed credentials needed: Users are migrated during their "normal" authentication process, so you don't need to import
+  credentials. Great if you don't have access to the hashed credentials.
+- No downtime: Both authentication systems operate simultaneously until the identity migration is completed.
+
+##### Drawbacks of graceful identity migration
+
+- Extended migration period: The process takes longer as users are migrated individually over time.
+- Increased complexity: Maintaining synchronization between two systems adds complexity to the migration process. (You need to 
+ensure identites are synchronized in both systems in case a rollback is required.) In addition, a graceful migration requires 
+additional coding effort to implement Ory's [password migration using a web hook](https://www.ory.com/docs/kratos/manage-identities/import-user-accounts-identities#password-migration-using-a-web-hook).
```

Comment:
I know I said this, but it's not a strict requirement. It depends on the legacy system if you need to do it this way.